build.xml

```diff
@@ -201,7 +201,7 @@ Class-Path: .</echo>
       <arg value="-webcache"/>
       <arg value="false"/>
       <arg value="-max-users"/>
-      <arg value="5"/>
+      <arg value="100"/>
```

Comment:
This is set low so it's easy to test the behaviour when the server is full. (Which eventually we should have browser tests for). Seems like the browser tests should invoke the jar directly with this higher value and no log.

Reply:
I think we do need a different way to start peergos since ant  also has the spurious "BUILD FAILED" message.